keyboards/ergotravel/keymaps/via/keymap.c

```diff
@@ -0,0 +1,48 @@
+#include QMK_KEYBOARD_H
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
```

Comment:
Optional, but you could use an `enum` here to manage the layer indexes.

```suggestion
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _ADJUST
};
```